package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amazon-connect-chatjs",
-  "version": "2.0.2",
+  "version": "2.1.0",
```

Comment:
can we update `package-lock.json` as well

Reply:
Yep!